Makefile

```diff
@@ -72,10 +73,11 @@ supportedsites:
 README.txt: README.md
 	pandoc -f markdown -t plain README.md -o README.txt
 
-youtube-dl.1: README.md
-	python devscripts/prepare_manpage.py >youtube-dl.1.temp.md
+youtube-dl.1: youtube_dl/*.py youtube_dl/*/*.py
```

Comment:
Current ```prepare_manpage.py``` only accepts the wide version of ```README.md``` (all options are in a single line). I generate a independent ```README-wide.md``` for generating the manpage only, so ```youtube-dl.1``` depends on the files ```README.md``` depends on rather than ```README.md```. I'll try to make the script more robust so that current ```README.md``` can be used to generate the manpage.